src/breadcrumbs/breadcrumbs.scss

```diff
@@ -59,5 +59,6 @@
 
   svg {
```

Comment:
so this is not targetting class, but still need to override display...

Reply:
The alignment fix is applicable to not just classname selectors.

![image](https://user-images.githubusercontent.com/9084735/131344074-dbb9d89e-59a2-4ff7-8a7c-14b66c5d7312.png)

Reply:
If I remove flex, then it jumps up a little bit